core/src/avm2/globals/flash/display/graphics.rs

```diff
@@ -951,11 +983,33 @@ fn draw_triangles_internal<'gc>(
     _uvt_data: Option<&Object<'gc>>,
     culling: TriangleCulling,
 ) -> Result<(), Error<'gc>> {
+    // FIXME Triangles should be drawn using non-zero winding rule.
+    //   When fixed, update output.expected.png of avm2/graphics_draw_triangles.
+    avm2_stub_method!(
+        activation,
+        "flash.display.Graphics",
+        "drawTriangles",
+        "winding behavior"
+    );
+
+    if _uvt_data.is_some() {
```

Comment:
This can be changed from `_uvt_data` to `uvt_data` now

Reply:
Fixed!